shub/image/test.py

```diff
@@ -105,4 +109,5 @@ def _run_docker_command(client, image_name, command):
                        stderr=True if statuscode else False,
                        stream=False, timestamps=False)
     utils.debug_log("{} results:\n{}".format(command, logs))
+    client.remove_container(container)
```

Comment:
@vshlapakov sorry, just realized that context manager is pointless, we just need try: finally: